src/presentations/indexable-presentation.php

```diff
@@ -334,6 +336,21 @@ public function generate_og_locale() {
 		return $this->og_locale_generator->generate( $this->context );
 	}
 
+	/**
+	 * Generates the open graph Facebook app ID.
+	 *
+	 * @return string The open graph Facebook app ID.
+	 */
+	public function generate_og_fb_app_id() {
+		$fb_app_id = $this->options_helper->get( 'fbadminapp', '' );
```

Comment:
You set the default (for when the option is not found) to an empty string. So I think you can just return it right away. No need for the check and empty string return that follow.

Reply:
Agreed and changed.

Reply:
Agreed and changed.